package.json

```diff
@@ -62,6 +66,9 @@
     "start": "node scripts/start.js",
     "build": "node scripts/build.js",
     "test": "node scripts/test.js",
+    "package:build": "tsc",
+    "package:watch": "tsc -w",
```

Comment:
These `package:*` tasks are awkwardly named, would love feedback on how this is handled.

Currently, the `build` and `start` tasks use webpack to compile and serve JS bundles in HTML files that demo the iframe message passing behavior and the example boost loader. But these don't compile the typescript into JS files and `.d.ts` declaration files as needed when another package imports from this package. So that's what `package:build` does, using `tsc`.

Wondering if that should be named just `build`, and the existing `build` and `start` could be named `bundle` and `serve` or something? Or maybe leave those and call the new script `tsc` or `compile`.

Reply:
Hmm, I do like how the tasks you mention, @tobek, are namespaced to `package` -- which I'm assuming ties to some sort of publish to npm or other package library. If they aren't tied to package publishing, maybe names like `compile` and `compile:watch` would be more descriptive.

Reply:
Ok cool, yeah on second pass I think `package:*` is actually pretty descriptive so I'll leave it at that (and `compile` could be ambiguous with `build`).